optuna/study.py

```diff
@@ -149,6 +149,15 @@ def trials(self) -> List[FrozenTrial]:
 
         return self.get_trials(deepcopy=True, states=None)
 
+    def _is_multi_objective(self) -> bool:
+        """Return True if the study has multi objectives.
+
+        Returns:
+            A boolean value indicates if the list `study.directions` has more than 1 element or not.
```

Comment:
```suggestion
            A boolean value indicates if `study.directions` has more than 1 element or not.
```

[IMO] We can remove `the list` here. What do you think?

Reply:
I agree with you. The word "directions" is itself plural.